.travis.yml

```diff
@@ -43,8 +79,12 @@ jobs:
   allow_failures:
     - os: osx
```

Comment:
Should we remove it as well? I think it was to mark the build as green fast without waiting for osx job. With this PR is not relevant.

Reply:
As the Marcos infra is less stable I'd leave this as is, so we can cancel the build if it hangs, for example.